### PR TITLE
[WIP] test PR

### PR DESCRIPTION
**Pull request by sandreenko**

### CODE REVIEW

File: src/jit/gentree.h [hunk 1]
```diff
@@ -5077,7 +5077,7 @@ struct Statement
     GenTree*       m_rootTree;      // root of the expression tree
     GenTree*       m_treeList;      // first node (for forward walks)
     InlineContext* m_inlineContext; // The inline context for this statement.
-    IL_OFFSETX     gtStmtILoffsx;   // instr offset (if available)
+    IL_OFFSETX     m_ILoffsx;       // instr offset (if available)
```
Comment (mikedn) [inline src/jit/gentree.h]: Any idea what the x suffix stands for? I'd just call it `m_ILoffset`.

**Comment by mikedn** [comment]
Nice! Just one big observation - it would highly preferable to stop making data members public and instead provide get/set/whatever accessors. Direct data member access should never be used in IR related classes - it makes it more difficult to understand where something is changed, it makes it impossible to validate changes (by adding asserts for example) and to add debugger breakpoints.

File: src/jit/gentree.h (hunk 1 above)
Comment (sandreenko) [reply src/jit/gentree.h]: > Any idea what the x suffix stands for? I'd just call it m_ILoffset.

It is done to dittinguish `IL_OFFSETX` from `IL_OFFSET`. `IL_OFFSETX` has two bytes as flags (`IL_OFFSETX_STKBIT`, `IL_OFFSETX_CALLINSTRUCTIONBIT` and I am planning to add `compilerAdded` there soon) and special values like `ICorDebugInfo::NO_MAPPING`, `ICorDebugInfo::PROLOG`, `ICorDebugInfo::EPILOG`.

**Pull request closed without merging**